chainguard-dev/edu
# Update incorrect URLs in tutorials

## Type of change
bug

### What should this PR do?
Fixes URLs in tutorials that link to missing pages.

### Why are we making this change?
Fix 404 links in tutorials

### What are the acceptance criteria? 
Any link in any of the affected pages should go to the correct page

### How should this PR be tested?
Check links using the netlify preview. I'll add a list of pages to check as a comment to this PR.
